{{cookiecutter.project_slug}}/README.rst

```diff
@@ -4,11 +4,16 @@
 
 {{ cookiecutter.project_short_description}}
 
-Try these notebooks on [pangeo.binder.io](http://binder.pangeo.io/): [![Binder](http://binder.pangeo.io/badge.svg)](http://binder.pangeo.io/v2/gh/pangeo-data/pangeo-example-notebooks/master)
+Try these notebooks on pangeo.binder.io_ : |Binder|
 
 See http://pangeo.io for more information.
 
 Features
 --------
 
-* TODO
\ No newline at end of file
+* TODO
+
+.. _pangeo.binder.io: http://binder.pangeo.io/
+
+.. |Binder| image:: http://binder.pangeo.io/badge.svg
+    :target: http://binder.pangeo.io/v2/gh/pangeo-data/pangeo-example-notebooks/master
```

Comment:
Thanks for the fix!

This link should actually be: http://binder.pangeo.io/v2/gh/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}/master

I actually made this change locally but it didn't make it into my commit for some reason.

Reply:
I thought so :). Do you want me to go ahead and fix the link for you as well?